Dockerfile

```
@@ -0,0 +1,99 @@
+FROM debian:12.9-slim
```

```suggestion
FROM debian:12.10-slim
```
There is a newer version of the base image available.